client/src/components/MapComponent.vue

```diff
@@ -54,13 +48,18 @@ export default {
       }
     },
     geolocate: function() {
```

Comment:
This is an uncommon syntax pattern for the Options API. If it aint broke dont fix it, but it may not be intuitive in a Vue2 sense